WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailFeaturedImageView.swift

```diff
@@ -0,0 +1,386 @@
+import UIKit
+
+protocol ReaderDetailFeaturedImageViewDelegate: class {
+    func didTapFeaturedImage(_ sender: CachedAnimatedImageView)
+}
+
+class ReaderDetailFeaturedImageView: UIView, NibLoadable {
+    struct Constants {
+        struct multipliers {
+            static let maxPortaitHeight: CGFloat = 0.70
+            static let maxPadPortaitHeight: CGFloat = 0.50
+            static let maxLandscapeHeight: CGFloat = 0.30
+        }
+    }
+
+    struct Styles {
+        static let startTintColor: UIColor = .white
+        static let endTintColor: UIColor = UIColor(light: .black, dark: .white)
```

Comment:
I guess you can replace that by using `.text`

Reply:
Done in: 51261fdb999244caa1d9e1148af2bca5ae5137a5